cirq/ops/raw_types.py

```diff
@@ -609,6 +609,24 @@ def _equal_up_to_global_phase_(self,
                                                   atol=atol)
 
 
+class VirtualTag():
```

Comment:
I am not sure that raw_types is the best place to accumulate tags.  

We should either make a virtual_tag.py or a tags.py to put this in.

Reply:
Opted for `tags.py` in lieu of other tag types.